src/main/java/com/stripe/model/Account.java

```diff
@@ -1025,6 +1025,9 @@ public void setFrontObject(File expandableObject) {
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class Settings extends StripeObject {
+    @SerializedName("bacs_debit_payments")
+    BacsDebitPayments bacsDebitPayments;
```

Comment:
This one doesn't have the `Settings` prefix because it's nested inside the `Settings` class as it should. It's the other ones we missed unfortunately and would need to change in a future major